exchanges/okgroup/okgroup.go

```diff
@@ -366,9 +368,22 @@ func (o *OKGroup) GetSpotFilledOrdersInformation(request GetSpotFilledOrdersInfo
 }
 
 // GetSpotMarketData Get the charts of the trading pairs. Charts are returned in grouped buckets based on requested granularity.
-func (o *OKGroup) GetSpotMarketData(request GetSpotMarketDataRequest) (resp GetSpotMarketDataResponse, _ error) {
+func (o *OKGroup) GetSpotMarketData(request *GetSpotMarketDataRequest) (resp GetSpotMarketDataResponse, err error) {
+	var assetNotSupported = errors.New("asset not supported")
```

Comment:
maybe we can shift this var outside of this scope? Also just quickly going over okex will support most of these but OKcoin won't, they just use spot so maybe a string switch for that case?